source/synthDrivers/espeak.py

```diff
@@ -106,10 +176,8 @@ def speak(self,speechSequence):
 			elif isinstance(item, CharacterModeCommand):
 				textList.append("<say-as interpret-as=\"characters\">" if item.state else "</say-as>")
 			elif isinstance(item, LangChangeCommand):
-				if langChanged:
-					textList.append("</voice>")
-				textList.append("<voice xml:lang=\"%s\">"%(item.lang if item.lang else defaultLanguage).replace('_','-'))
-				langChanged=True
+				self._handleLangChangeCommand(item, textList, langChanged)

```

Comment:
Since the return value isn't used and `textList` is only ever appended to, this could be made more functional. Return a string that is appended to the `textList` here.

Reply:
Fixed in 5c3388b